pkg/controller/user/user.go

```diff
@@ -31,24 +32,33 @@ import (
 
 // Controller manages users
 type Controller struct {
-	cacher cache.Cacher
-	client *auth.Client
-	config *config.ServerConfig
-	db     *database.Database
-	h      *render.Renderer
-	logger *zap.SugaredLogger
+	cacher     cache.Cacher
+	fbInternal *iFB.Client
+	client     *auth.Client
+	config     *config.ServerConfig
+	db         *database.Database
+	h          *render.Renderer
+	logger     *zap.SugaredLogger
 }
 
 // New creates a new controller for managing users.
-func New(ctx context.Context, client *auth.Client, cacher cache.Cacher, config *config.ServerConfig, db *database.Database, h *render.Renderer) *Controller {
+func New(
+	ctx context.Context,
+	fbInternal *iFB.Client,
```

Comment:
So I actually have this dream of moving everything to `internal/firebase`, but that's for another day 😄